package/MDAnalysis/coordinates/chain.py

```diff
@@ -299,7 +299,8 @@ def __init__(self, filenames, skip=1, dt=None, continuous=False,
 
         # calculate new start_frames to have a time continuous trajectory.
         if continuous:
-            check_allowed_filetypes(self.readers, ['XTC', 'TRR', "LAMMPSDUMP"])
+            check_allowed_filetypes(self.readers, ['XTC', 'TRR', "LAMMPSDUMP",
+                                                   "TRC"])
```

Comment:
Could you please make all strings here use the same quotes, just for aesthetics :-).

Reply:
Yes, I agree .)